pkgs/by-name/st/stardust-xr-server/package.nix

```diff
@@ -0,0 +1,101 @@
+{
+  lib,
+  fetchFromGitHub,
+  writeShellApplication,
+  common-updater-scripts,
+  coreutils,
+  git,
+  nix,
+  rustPlatform,
+  openxr-loader,
+  libGL,
+  mesa,
+  xorg,
+  fontconfig,
+  libxkbcommon,
+  libclang,
+  cmake,
+  cpm-cmake,
+  pkg-config,
+  llvmPackages,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "stardust-xr-server";
+  version = "0.44.1";
+
+  src = fetchFromGitHub {
+    owner = "stardustxr";
+    repo = "server";
+    rev = "refs/tags/${version}";
+    hash = "sha256-sCatpWDdy7NFWOWUARjN3fZMDVviX2iV79G0HTxfYZU=";
+  };
+
+  cargoLock = {
+    lockFile = ./Cargo.lock;
+    outputHashes = {
+      "smithay-0.3.0" = "sha256-dxAgTGW+xxnL+vA6j2Ng02F1zt/Y5VaSxP9xg8jfMy8=";
+      "stardust-xr-0.14.1" = "sha256-fmRb46s0Ec8wnoerBh4JCv1WKz2of1YW+YGwy0Gr/yQ=";
+    };
+  };
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+    llvmPackages.libcxxClang
+  ];
+  buildInputs = [
+    openxr-loader
+    libGL
+    mesa
+    xorg.libX11
+    fontconfig
+    libxkbcommon
+  ];
+
+  LIBCLANG_PATH = "${libclang.lib}/lib";
```

Comment:
Try `rustPlatform.bindgenHook` (add to `nativeBuildInputs`), if that works remove all things with `clang`.

Reply:
That works. Thanks!